test/formatDate.spec.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from "chai";
+import React from "react";
+import { shallow } from "enzyme";
+import Globalize from "globalize";
+import FormatDate from "../dist/date";
+
+Globalize.load(
+    require( "cldr-data/main/en/ca-gregorian" ),
+    require( "cldr-data/main/en/timeZoneNames" ),
+    require( "cldr-data/main/en/numbers" ),
+    require( "cldr-data/supplemental/likelySubtags" ),
+    require( "cldr-data/supplemental/timeData" ),
+    require( "cldr-data/supplemental/weekData" )
+);
+
+describe("formatDate Component", () => {
+    it("renders as a <span>", () => {
+        const wrapper = shallow(<FormatDate locale="en" pattern="GyMMMd">{new Date()}</FormatDate>);
+        expect(wrapper.type()).to.eql("span");
+    });
+
+    it("formats date as 1/1/2016", () => {
+        const wrapper = shallow(<FormatDate locale="en" pattern="GyMMMd">{new Date("Jan 01 2016")}</FormatDate>);

```

Comment:
About the lines you have removed, they are there because of esperanto (kind of a hack for esperanto to correctly build it for AMD), we'll need to investigate how to fix this.